core/templates/dev/head/css/oppia.css

```diff
@@ -3951,3 +3951,10 @@ span.oppia-issues-learner-action {
   width: 100%;
   border: 1px solid black;
 }
+
+/*To fix the shifting of tooltip when text is too long*/
+
+div.tooltip-inner{
```

Comment:
Style: there should be a space before {. See other rules in this file.

/cc @apb7, this should be caught automatically.

Reply:
Hm. We do have a check for this. This commit has been made using GitHub's online editor as far as I can see.